.storybook/test-data.js

```diff
@@ -102,6 +102,43 @@ const state = {
       swapsFeatureIsLive: false,
       swapsQuoteRefreshTime: 60000,
     },
+    snaps: {
+      "foo": {
+        name: 'foo',
```

Comment:
this will have `.id` and `.proposedName`? @rekmarks

Reply:
Nope! Ref:

```json
{
  "snapStates": {},
  "snaps": {
    "local:http://localhost:8080/": {
      "enabled": true,
      "id": "local:http://localhost:8080/",
      "initialPermissions": {
        "snap_confirm": {}
      },
      "manifest": {
        "description": "An example MetaMask Snap.",
        "initialPermissions": {
          "snap_confirm": {}
        },
        "manifestVersion": "0.1",
        "proposedName": "MetaMask Example Snap",
        "repository": {
          "type": "git",
          "url": "https://github.com/MetaMask/snaps-skunkworks.git"
        },
        "source": {
          "location": {
            "npm": {
              "filePath": "dist/bundle.js",
              "iconPath": "images/icon.svg",
              "packageName": "@metamask/example-snap",
              "registry": "https://registry.npmjs.org/"
            }
          },
          "shasum": "3lEt0yUu080DwV78neROaAAIQWXukSkMnP4OBhOhBnE="
        },
        "version": "0.6.0"
      },
      "permissionName": "wallet_snap_local:http://localhost:8080/",
      "sourceCode": "(...)",
      "status": "stopped",
      "svgIcon": "<svg>...</svg>",
      "version": "0.6.0"
    }
  }
}
```

Reply:
And `SubjectMetadata`:

```json
{
  "subjectMetadata": {
    "http://localhost:8080": {
      "extensionId": null,
      "iconUrl": null,
      "name": "Hello, Snaps!",
      "origin": "http://localhost:8080",
      "subjectType": "website"
    },
    "https://metamask.github.io": {
      "extensionId": null,
      "iconUrl": null,
      "name": "Snaps Iframe Execution Environment",
      "origin": "https://metamask.github.io",
      "subjectType": "website"
    },
    "local:http://localhost:8080/": {
      "extensionId": null,
      "iconUrl": null,
      "name": "MetaMask Example Snap",
      "origin": "local:http://localhost:8080/",
      "subjectType": "snap",
      "svgIcon": "<svg>...</svg>",
      "version": "0.6.0"
    }
  }
}
```